### PR TITLE
Update to master

### PR DESCRIPTION
**Pull request by fgimenez**
Towards https://github.com/giantswarm/giantswarm/issues/2023

Pulls latest changes from master

### CODE REVIEW

**Comment by rossf7** [comment]
The installation name also needs to be passed to the legacy resource so it can tag the VPC. Just pushed a fix.

File: service/service.go [hunk 1]
```diff
@@ -254,7 +276,7 @@ func New(config Config) (*Service, error) {
 		frameworkConfig.BackOff = frameworkBackOff
 		frameworkConfig.InitCtxFunc = initCtxFunc
 		frameworkConfig.Logger = config.Logger
-		frameworkConfig.Resources = resources
+		frameworkConfig.ResourceRouter = framework.NewDefaultResourceRouter(resources)
```
Comment (rossf7) [inline service/service.go]: Here we use the new ResourceRouter. It's not fully implemented yet so all resources are returned.

File: vendor/github.com/giantswarm/k8scloudconfig/v_0_1_0/master_template.go [hunk 2]
```diff
@@ -854,11 +855,12 @@ write_files:
           serviceAccountName: nginx-ingress-controller
           containers:
           - name: nginx-ingress-controller
-            image: {{.Cluster.Kubernetes.IngressController.Docker.Image}}
+            image: quay.io/kubernetes-ingress-controller/nginx-ingress-controller:0.9.0-beta.17
```
Comment (rossf7) [inline vendor/github.com/giantswarm/k8scloudconfig/v_0_1_0/master_template.go]: Can you confirm we want these ingress changes to be vendored?

Comment (fgimenez) [reply vendor/github.com/giantswarm/k8scloudconfig/v_0_1_0/master_template.go]: Very good point, checked with @puja108 and it's ok to have this in place for new clusters.

**Pull request merged**